code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -414,72 +429,86 @@
 				sp.start()
 				S.Weaken(6)
 		for(var/obj/structure/window/W in T.contents)
-			W.hit(rand(50,100))
+			W.hit(rand(80, 100))
 
 
 
-/obj/effect/proc_holder/spell/aoe_turf/drain_thralls
-	name = "Drain Thralls"
-	desc = "Damages nearby thralls, draining their life and healing yourself."
+/obj/effect/proc_holder/spell/aoe_turf/drainLife
+	name = "Drain Life"
+	desc = "Damages nearby humans, draining their life and healing your own wounds."
 	panel = "Shadowling Abilities"
 	range = 3
 	charge_max = 100
 	clothes_req = 0
-	var/thralls_drained = 0
-	var/list/nearby_thralls = list()
+	var/targetsDrained
+	var/list/nearbyTargets
 
-/obj/effect/proc_holder/spell/aoe_turf/drain_thralls/cast(list/targets)
-	thralls_drained = 0
-	nearby_thralls = list()
+/obj/effect/proc_holder/spell/aoe_turf/drainLife/cast(list/targets, var/mob/living/carbon/human/U = usr)
+	targetsDrained = 0
+	nearbyTargets = list()
 	for(var/turf/T in targets)
 		for(var/mob/living/carbon/M in T.contents)
-			if(is_thrall(M))
-				thralls_drained++
-				nearby_thralls.Add(M)
-				M << "<span class='warning'>You feel a curious draining sensation and a wave of exhaustion washes over you.</span>"
-		for(var/mob/living/carbon/M in nearby_thralls)
-			nearby_thralls.Remove(M) //To prevent someone dying like a zillion times
-			M.take_organ_damage(25/thralls_drained,25/thralls_drained) //For every nearby thrall, the damage to each is reduced - 1 thrall = 50 for him, 2 thralls = 25 for each, etc.
-			usr << "<span class='deadsay'>You draw the life from [M] to heal your wounds.</span>"
-	if(thralls_drained)
-		var/mob/living/carbon/U = usr
-		U.heal_organ_damage(25, 25)
-	else
+			targetsDrained++
+			nearbyTargets.Add(M)
+		for(var/mob/living/carbon/M in nearbyTargets)
+			nearbyTargets.Remove(M) //To prevent someone dying like a zillion times
```

Comment:
I don't see any way it could be, unless someone managed to be put in the target list twice.  Am I missing something, @Xhuis?